--- a/csrc/libtorch_stable/quantization/w8a8/cutlass/scaled_mm_entry.cu
+++ b/csrc/libtorch_stable/quantization/w8a8/cutlass/scaled_mm_entry.cu
@@ -1,3 +1,5 @@
+#define ENABLE_CUTLASS_MOE_SM120 1
+#define ENABLE_SCALED_MM_SM120 1
 #include <cudaTypedefs.h>
 
 #include <torch/csrc/stable/tensor.h>
@@ -84,9 +86,7 @@
     const std::optional<torch::stable::Tensor>& blockscale_offsets,
     const bool is_gated);
 
-#if (defined(ENABLE_CUTLASS_MOE_SM90) && ENABLE_CUTLASS_MOE_SM90) ||   \
-    (defined(ENABLE_CUTLASS_MOE_SM100) && ENABLE_CUTLASS_MOE_SM100) || \
-    (defined(ENABLE_CUTLASS_MOE_SM120) && ENABLE_CUTLASS_MOE_SM120)
+#if 1
 void get_cutlass_moe_mm_data_caller(
     const torch::stable::Tensor& topk_ids,
     torch::stable::Tensor& expert_offsets,
@@ -232,7 +232,7 @@
   int32_t version_num = get_sm_version_num();
 
 #if defined ENABLE_SCALED_MM_SM120 && ENABLE_SCALED_MM_SM120
-  if (version_num >= 120) {
+  if (version_num >= 120 || version_num == 121) {
     cutlass_scaled_mm_sm120(c, a, b, a_scales, b_scales, bias);
     return;
   }
@@ -311,9 +311,7 @@
   // This function currently gets compiled only if we have a valid cutlass moe
   // mm to run it for.
   int32_t version_num = get_sm_version_num();
-#if (defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90) ||   \
-    (defined ENABLE_CUTLASS_MOE_SM100 && ENABLE_CUTLASS_MOE_SM100) || \
-    (defined ENABLE_CUTLASS_MOE_SM120 && ENABLE_CUTLASS_MOE_SM120)
+#if 1
   get_cutlass_moe_mm_data_caller(topk_ids, expert_offsets, problem_sizes1,
                                  problem_sizes2, input_permutation,
                                  output_permutation, num_experts, n, k,
@@ -333,9 +331,7 @@
     torch::stable::Tensor& problem_sizes1,
     torch::stable::Tensor& problem_sizes2, const int64_t n, const int64_t k,
     const bool swap_ab) {
   int32_t version_num = get_sm_version_num();
-#if (defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90) ||   \
-    (defined ENABLE_CUTLASS_MOE_SM100 && ENABLE_CUTLASS_MOE_SM100) || \
-    (defined ENABLE_CUTLASS_MOE_SM120 && ENABLE_CUTLASS_MOE_SM120)
+#if 1
   get_cutlass_moe_mm_problem_sizes_from_expert_offsets_caller(
       expert_first_token_offset, problem_sizes1, problem_sizes2, n, k, swap_ab);
   return;
@@ -357,9 +353,7 @@
   // This function currently gets compiled only if we have a valid cutlass moe
   // mm to run it for.
   int32_t version_num = get_sm_version_num();
-#if (defined ENABLE_CUTLASS_MOE_SM90 && ENABLE_CUTLASS_MOE_SM90) ||   \
-    (defined ENABLE_CUTLASS_MOE_SM100 && ENABLE_CUTLASS_MOE_SM100) || \
-    (defined ENABLE_CUTLASS_MOE_SM120 && ENABLE_CUTLASS_MOE_SM120)
+#if 1
   get_cutlass_batched_moe_mm_data_caller(expert_offsets, problem_sizes1,
                                          problem_sizes2, expert_num_tokens,
                                          num_local_experts, padded_m, n, k);
